### ISSUE TITLE
Table widths don't account for type B font

### ISSUE THREAD

**Issue by pmccarren** [Klemen1337/node-thermal-printer#36]
Hello!

**TLDR; What's broken?**
The table generator does not account for the extra line length when using `Font B`

**More Info**
When `Font B` is used each line is an extra 15 characters wide and the line height remains the same. The problem is that the table generator does not account for the extra line length.

I'm using the Epson TM-T88V, so the 15 character difference might be different on other printers (maybe even less?)

From what it looks like, fixing this would require:
- Width configuration for each of the two font types
- Tracking which font mode we are in (note: setting text to `normal` resets font type)
- Adjusting table width and padding calculation to use the correct width configuration.

I wanted to get this tracked in an issue and will happily knock this out in a PR, but probably won't have time for several weeks though.

**Comment by Klemen1337** [comment]
Hi @pmccarren good point. I never considered that because i was only working with type A font.
